spyder/app/mainwindow.py

```diff
@@ -288,6 +288,7 @@ def signal_handler(signum, frame=None):
                 pass
         else:
             signal.signal(signal.SIGTERM, signal_handler)
+            signal.signal(signal.SIGINT, signal_handler)

```

Comment:
Why is this here?

Reply:
https://github.com/spyder-ide/spyder/issues/5114#issuecomment-327024883

It make possible to kill spyder using `Ctrl+C`, without this change It will still raise the report error message box when pressing `Ctrl+C`

Reply:
Please remove this. We need to discuss it in another PR.

Reply:
@ccordoba12 why? I think it is important for people to be able to quit spyder on the terminal using `Ctrl+C`

Reply:
When Spyder freezes for whatever reason, it's very useful to press `Ctrl+C` in the terminal and get the traceback of where it was freezed.

Reply:
But, the users will continue reporting false positives (now random), of when they tried to close spyder with `Ctrl+C`

Reply:
But that only happens when users press `Ctrl+C` in a terminal, and I doubt there are many of them.

In any case, I think this needs more discussion and is not related to this PR. So please remove it and open a new PR with change only.